packages/01-host/public/index.html

```diff
@@ -1,10 +1,25 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <script src="http://localhost:3002/remoteEntry.js"></script>
-    <script src="http://localhost:3003/remoteEntry.js"></script>
+    <script type="systemjs-importmap">
+      {
+        "imports": {
```

Comment:
The import map

Reply:
Don't we need an org name here in the app names

Reply:
org names are a best practice but are not required.

Reply:
Ok, just wondering